app/services/post_status_service.rb

```diff
@@ -41,6 +41,7 @@ def call(account, text, in_reply_to = nil, **options)
     DistributionWorker.perform_async(status.id)
 
     status.local_only = status.marked_local_only?
+    status.save!
```

Comment:
mmmmmm now that i think about this actually this means we are saving the status twice right?

it's saved once when it's created above and again here

ideally we set the flag when it's created in the first place...

Reply:
oh yeah! good point :smiley: